### PR TITLE
Use Nullness checker to annotate KeyStore.java.

### PR DESCRIPTION
**Pull request by xingweitian**
Related PR:
- typetools/annotated-libraries#38
- typetools/jdk#20

### CODE REVIEW

File: checker/build.gradle [hunk 1]
```diff
@@ -2,7 +2,7 @@ ext {
     // See instructions for updating jdk8.jar at
     // https://github.com/typetools/annotated-libraries/blob/master/README.md .
     // Set jdkShaHash to 'local' to use a locally-built version of jdk8.jar .
-    jdkShaHash = 'd7f53f36771f8007e617c52aa0e2dcc6213533d7'
+    jdkShaHash = '0e19e4268'
```
Comment (mernst) [inline checker/build.gradle]: Please use a full SHA hash, not an abbreviated one.

**Comment by xingweitian** [comment]
@mernst I have already changed to the correct jdkShaHash and travis has been passed.

Here is my pull request on typetools/jdk: typetools/jdk#20
And the corresponding jdk8 pull request: typetools/annotated-libraries#38

**Comment by mernst** [comment]
Thanks for the annotations.

There were problems with this pull request and the associated pull request https://github.com/typetools/jdk/pull/20.

 * `@Nullable char [] varname` is a declaration of an array of `@Nullable char`, which makes no sense because `char` is primitive.  I think you meant `char @Nullable [] varname`, which declares a nullable array of `char`.
 * There were missing annotations, which I discovered by grepping for "null" in the documentation.  When annotating a file, it is important to annotate the whole thing, not just a few methods.  Otherwise, it is very confusing for people later to use a partially-annotated file or to debug why the annotations are not consistently present.
 * You did not add `@AnnotatedFor` annotations to the classes.

Please see the changes that I made to both pull requests.  In future pull requests, could you address those items?

I think this is ready to be merged now (though you also assigned @wmdietl).  Thanks again.

**Comment by xingweitian** [comment]
> Thanks for the annotations.
> 
> There were problems with this pull request and the associated pull request [typetools/jdk#20](https://github.com/typetools/jdk/pull/20).
> 
> * `@Nullable char [] varname` is a declaration of an array of `@Nullable char`, which makes no sense because `char` is primitive.  I think you meant `char @Nullable [] varname`, which declares a nullable array of `char`.
> * There were missing annotations, which I discovered by grepping for "null" in the documentation.  When annotating a file, it is important to annotate the whole thing, not just a few methods.  Otherwise, it is very confusing for people later to use a partially-annotated file or to debug why the annotations are not consistently present.
> * You did not add `@AnnotatedFor` annotations to the classes.
> 
> Please see the changes that I made to both pull requests. In future pull requests, could you address those items?
> 
> I think this is ready to be merged now (though you also assigned @wmdietl). Thanks again.

Thanks a lot for the helpful advice! Sorry for the missed and wrong usage of annotations, I will do better in the next time.

**Comment by wmdietl** [comment]
@xingweitian Thanks for the specifications and @mernst thanks for the review!

**Pull request merged**